tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -11,12 +11,32 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi qw(save_screenshot get_required_var);
+use testapi qw(save_screenshot get_required_var get_var);
 
 sub run {
     my @scc_addons = grep($_, split(/,/, get_required_var('SCC_ADDONS')));
     $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
     save_screenshot;
+    my $timeout = 60 * get_var('TIMEOUT_SCALE', 1);
+    $testapi::distri->wait_registration_common_regcode_finished({timeout => $timeout, interval => 2, message => 'Page to insert module registration code did not appear'});
+    foreach my $addon (@scc_addons) {
+        if ($addon eq 'we' && get_var('SCC_REGCODE_WE', '')) {
```

Comment:
it would fail anyway, it is good practice, but it is rare settings are not there, means that the configuration is completely mess up and the test might not need to care for such a bad situation, but more for testing functionality not infrastructure/test configuration.